### DISCUSSION TITLE
Not all beatmaps are available in lazer?

### DISCUSSION THREAD

**Discussion by Shirou27** [ppy/osu#22312] [Q&A] [answered]
When i go to osu.ppy.sh beatmap list, there's many beatmaps there especially anime opening and ending songs but in osu!lazer there's only a few. 

The beatmaps in lazer beatmap list is not interesting to listen and play that's why i can't download a beatmap anymore and i thought the beatmap makers reduced significantly, but I'm glad that i found out that its not. the list just got separated from the real beatmap list.

i wonder why?

![Screenshot_2023-01-21-12-26-24-882_com opera browser](https://user-images.githubusercontent.com/105548541/213843530-2f2ed96d-d6f0-4d14-a6c6-d08c5cb7946d.jpg)

![Screenshot_2023-01-21-12-27-16-415_sh ppy osulazer](https://user-images.githubusercontent.com/105548541/213843534-511687a7-2a86-413e-9c74-12b7fe1b347b.jpg)

**Comment by Flamiii** [comment]
In the lazer beatmap listing, you have 'Featured artists' ticked.

**Comment by uqsme** [comment, marked as answer]
Starting from version `2022.1227.0`, the `featured artists` filter is turned on by default.
You need to turn it down **every time you launch your game**, click the yellow `featured artists` tag and **hold on** `I understand`.

![1674279702_FE9251B59CE4415E81E32C45660C0167](https://user-images.githubusercontent.com/41455159/213845519-5ad56e11-982f-44ee-8a27-c3621f5d1872.jpeg)
